### ISSUE TITLE
Can't pull from a chest anymore

### ISSUE THREAD

**Issue by ShaneCourtrille** [sayterdarkwynd/FrackinUniverse#553]
Got the latest version tonight and my storage setup no longer works.  I have a storage adapter red -> router blue -> router red -> repeater blue -> repeater red to chest blue.

I've also tried removing the repeater and same problem.  Oddly when I broke the router to try a full reset it dropped some canvas? *shrug*

**Comment by ShaneCourtrille** [comment]
Stopped/Started the server and things are working now.  As a side note when I first started half my storage buses were missing.  Looks like just an update hiccup.

**Comment by ShaneCourtrille** [comment]
I had thought the problem went away with a server restart but I had forgotten I had manually emptied the input chest.  Still has the problem.

**Comment by Kherae** [comment]
in most cases an 'active' wire connection (bright red) means it recognizes it's connected to something. more or less a semi-useless debug indicator since I didn't actually flesh it out that much. recent update broke the storage bridges a tad because they have new sprites. make thing go boom. if things stop working, generally it's best to pick the whole network up and re-place it. see if that fixes it. if not, tell me if the log has anything. if so..share.

**Comment by ShaneCourtrille** [comment]
Already took it all apart and re-created it in different ways and none work.  Do you have a link for where I can find out info on the log you're talking about?

**Comment by ShaneCourtrille** [comment]
Re-created it on a new world and it works.  I'll test more to see if I can figure out whats wrong with the original.

**Comment by Kherae** [comment]
if you cant find it or dont know what to look for, just give the whole log. pastebin ftw.

**Comment by ShaneCourtrille** [comment]
So it happened again on a brand new setup  and when I broke the "item transferrence device" that pulled from the input chest and placed things into the storage chests I again got an item dropped.  The weird thing is that the item I got was something I'm 90% sure I didn't place into the input chest.  In fact I'm pretty sure it's a dupe item since I've only ever made two of this item and both are accounted for already.

This time when I rebuilt it all worked fine.
